db/models/topics.models.js

```diff
@@ -157,6 +157,19 @@ exports.removeCommentAtId = (comment_id) => {
     } else {
       return PromResults[1].rows;
     }
-});
+  });
 };
 
+exports.retrieveAllUsers = () => {
+  return db
+    .query(
+      `
+        SELECT * FROM users `
+    )
+    .then((users) => {
+      return users.rows;
+    })
+    .catch((err) => {
+      next(err);
+    });
```

Comment:
Shouldn't need a catch in our model, our controllers will catch any errors from this query.